examples/unet_segmentation_3d.py

```diff
@@ -34,16 +37,13 @@
 from monai.data.synthetic import create_test_image_3d
 from monai.handlers.utils import stopping_fn_from_metric
 
-# assumes the framework is found here, change as necessary
-sys.path.append("..")
-
-config.print_config()
+monai.config.print_config()
 
 # Create a temporary directory and 50 random image, mask paris
 tempdir = tempfile.mkdtemp()
 
 for i in range(50):
-    im, seg = create_test_image_3d(256, 256, 256)
+    im, seg = create_test_image_3d(128, 128, 128)
```

Comment:
this function by default generates `seg` with 5 foreground classes, I think we need 
```im, seg = create_test_image_3d(128, 128, 128, num_seg_classes=1)```
so that it's consistent with the network's `num_classes=1` setting

Reply:
Fixed